01_fib/ovlords/main.go

```diff
@@ -0,0 +1,32 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+var out io.Writer = os.Stdout
+
+func fib(n int) {
+
+	if n > 0 {
+
+		currentNumber, previousNumber, scratchNumber := 1, 0, 0
```

Comment:
Go programs generally don't put types on the end of variable names, so this would be:
```
current, previous, scratch := 1, 0, 0
```